release-calendar/webpack.client.dev.config.js

```diff
@@ -17,6 +17,8 @@
 const path = require('path');
 const HtmlWebPackPlugin = require('html-webpack-plugin');
 const Dotenv = require('dotenv-webpack');
+const MomentTimezoneDataPlugin = require('moment-timezone-data-webpack-plugin');
+const webpack = require('webpack');
```

Comment:
Adjusting for timezone should not happen during bundle compilation, because it depends on who's doing the compiling. I don't think it this belongs in webpack.

Reply:
Note for later: we talked offline. We decided to keep them. Cite this for reasons for the empty module https://github.com/moment/moment/issues/2979